app/helpers/tasklist_helper.rb

```diff
@@ -0,0 +1,54 @@
+module TasklistHelper
+  STATUSES = {
+    :not_started => "Not started",
+    :cannot_start_yet =>  "Cannot start yet",
+    :completed => "Completed",
+    :in_progress => "In progress"
+  }
```

Comment:
maybe you could add `.freeze` to the end? I imagine these will be constant

Reply:
Yup! Nice one!